tests/test_integrated_channels/test_sap_success_factors/test_client.py

```diff
@@ -242,7 +233,7 @@ def test_expired_access_token(self):
         sap_client = SAPSuccessFactorsAPIClient(self.enterprise_config)
         actual_response = sap_client.create_course_content(payload)
         assert actual_response == expected_response
-        assert len(responses.calls) == 3  # pylint: disable=no-member
-        assert responses.calls[0].request.url == self.url_base + self.oauth_api_path  # pylint: disable=no-member
-        assert responses.calls[1].request.url == self.url_base + self.oauth_api_path  # pylint: disable=no-member
-        assert responses.calls[2].request.url == self.url_base + self.course_api_path  # pylint: disable=no-member
```

Comment:
@UmanShahzad Thanks for cleaning up these `pylint` exceptions!

Reply:
:+1: